### PR TITLE
[1LP][RFR] Fix to RHSM/SAT6 registrations

### PR DESCRIPTION
**Pull request by lcouzens**
Update to RHSM/SAT6 validation and registration testing. Bringing it up to the latest build versions.

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/configure/test_register_appliance.py [hunk 1]
```diff
@@ -33,20 +30,19 @@ def pytest_generate_tests(metafunc):
...
-            # We cannot validate against Satellite 5
-            if metafunc.function.__name__ == 'test_rh_creds_validation' and reg_method == 'sat5':
-                continue
 
             reg_data = all_reg_data.get(reg_method)
             if not reg_data or not reg_data.get('test_registration', False):
                 continue
 
-            proxy_data = conf.cfme_data['redhat_updates'].get('http_proxy', False)
+            proxy_data = conf.cfme_data.get('redhat_updates', {}).get('http_proxy', False)
```
Comment (izapolsk) [inline cfme/tests/configure/test_register_appliance.py]: :+1:

File: utils/appliance/__init__.py [hunk 2]
```diff
@@ -204,6 +204,30 @@ def get(self, cls, *args, **kwargs):
...
+        self.ssh_client.run_command('subscription-manager unregister')
+        self.ssh_client.run_command('subscription-manager clean')
+
+    def sat6_unregister(self):
+        """ unregisters an appliance from SAT6 subscription """
+        self.ssh_client.run_command('subscription-manager remove --all')
+        self.ssh_client.run_command('subscription-manager unregister')
+        self.ssh_client.run_command('subscription-manager clean')
+        self.ssh_client.run_command('mv -f /etc/rhsm/rhsm.conf.kat-backup /etc/rhsm/rhsm.conf')
+        self.ssh_client.run_command('rpm -qa | grep katello-ca-consumer | xargs rpm -e')
```
Comment (izapolsk) [inline utils/appliance/__init__.py]: wouldn't it be easier to use yum or dnf here ?

Comment (psav) [inline utils/appliance/__init__.py]: I think rpm-qa is quicker and platform agnostic ;)

**Comment by dajoRH** [comment]
Lint report for commit e101cd5e902b38a35ed082205574dcbd45b84f8e:

`cfme/base/ui.py`:
- :bangbang: Line 609:1: **E302** *expected 2 blank lines, found 1*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**